src/frontend/apps/lti_site/components/LocalizedTimedTextTrackUpload/LanguageSelect/index.tsx

```diff
@@ -41,9 +41,16 @@ export const LanguageSelect = ({
     [intl, messages],
   );
 
-  const { choices, getChoices } = useTimedTextTrackLanguageChoices(
-    (state) => state,
+  const { data } = useFetchTimedTextTrackLanguageChoices();
+  const choices = useMemo(
+    () =>
+      data?.actions.POST.language.choices?.map((choice) => ({
+        label: choice.display_name,
+        value: choice.value,
+      })),
+    [data],
```

Comment:
data is too general, useMemo will refresh too often unnecessarily.
`data?.actions.POST.language.choices`